source/auth/auth.rst

```diff
@@ -13,7 +13,7 @@ Driver Authentication
 :Status: Accepted
 :Type: Standards
 :Minimum Server Version: 2.6
-:Last Modified: 2020-01-13
+:Last Modified: 2020-01-16
```

Comment:
Spec version should be bumped for this. I think it's debatable if this warrants a bump to the patch or minor version. I'd be OK with a patch bump to 1.8.3 since it's more of a clarification than new functionality.

Reply:
I have bumped the version to 1.8.3.

Reply:
I see you added 1.8.3 to the "Version History" section below, but I was referring to the "Spec Version" a few lines up.

Side note: a "Version History" section for this spec seems inconsistent with the "Changelog" section in most of our other specs, which typically just has a date and a short description of the change. This isn't something to address now, though.

Reply:
I have updated the Spec Version.